test/integration/ggrc/converters/test_export_csv.py

```diff
@@ -491,6 +491,19 @@ def test_priority_tip(self, model):
     self.assertIn("Allowed values are:\n{}".format('\n'.join(
         constants.AVAILABLE_PRIORITIES)), response.data)
 
+  @ddt.data("Assessment", "Audit", "AssessmentTemplate")
+  def test_archived_attributes(self, model):
+    """Test to check import values for archived objects"""
```

Comment:
Please fix the type in the docstring.
```suggestion
    """Test to check export values for archived objects"""
```